src/mono/browser/build/WasmApp.InTree.props

```diff
@@ -29,7 +29,7 @@
   <ItemGroup>
     <_MonoRuntimeComponentDontLink Include="libmono-component-debugger-stub-static.a" />
     <!-- FIXME: This will exclude the diagnostics component if tracing is not on and threads are not on.  Which means that if you turn on threading, you will get diagnostics.  Is this what we want? -->
-    <_MonoRuntimeComponentDontLink Include="libmono-component-diagnostics_tracing-static.a" Condition="'$(FeatureWasmPerfTracing)' != 'true' and $(FeatureWasmThreads) != 'true'"/>
+    <_MonoRuntimeComponentDontLink Include="libmono-component-diagnostics_tracing-static.a" Condition="'$(FeatureWasmPerfTracing)' != 'true' and $(FeatureWasmManagedThreads) != 'true'"/>
```

Comment:
I assume this is linking in a stub, instead of real (larger) component.
I think we could keep only `'$(FeatureWasmPerfTracing)' != 'true'` and remove the other one.